bin/notificators/hard_threshold_notificator.py

```diff
@@ -45,11 +45,13 @@ def send_slack_notification(self, query, is_send_to_user=True, is_send_to_admin_
             user=query.executing_user
         )
 
-        text = self.config.SLACK_HARD_THRESHOLD_MESSAGE.format(**params)
+        text_1 = self.config.SLACK_HARD_THRESHOLD_MESSAGE_1.format(**params)
```

Comment:
🔴 
This change will fail tests. 
You haven't changed the variable name across the repository, e.g.: https://github.com/Wikia/athena_alerter/blob/2afa751c911085e8ea53f7b265c778c9f216d6aa/tests/test_notification.py#L27

Reply:
👍 I reverted the variable name to the original as recommended in the next comment.

Reply:
🔴 
Please fix tests so they pass:

```
make test

EE......
======================================================================
ERROR: test_handle_batch_event_test_separate_thresholds (tests.notificators.test_hard_threshold_notificator.TestHardThresholdNotificator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mikolajbalcerek/.pyenv/versions/3.9.4/lib/python3.9/unittest/mock.py", line 1337, in patched
    return func(*newargs, **newkeywargs)
  File "/Users/mikolajbalcerek/Dev/athena_alerter/tests/notificators/test_hard_threshold_notificator.py", line 46, in test_handle_batch_event_test_separate_thresholds
    sut.handle_batch_event(events)
  File "/Users/mikolajbalcerek/Dev/athena_alerter/bin/notificators/hard_threshold_notificator.py", line 25, in handle_batch_event
    self.handle_single_event(body=record['body'])
  File "/Users/mikolajbalcerek/Dev/athena_alerter/bin/notificators/hard_threshold_notificator.py", line 35, in handle_single_event
    self.send_slack_notification(query, is_send_to_user, is_send_to_admin_channel)
  File "/Users/mikolajbalcerek/Dev/athena_alerter/bin/notificators/hard_threshold_notificator.py", line 52, in send_slack_notification
    self.send_slack_to_channel(text + text_additional_1)
TypeError: can only concatenate str (not "Mock") to str

======================================================================
ERROR: test_handle_batch_event_user_and_channel_value_above_threshold (tests.notificators.test_hard_threshold_notificator.TestHardThresholdNotificator)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mikolajbalcerek/.pyenv/versions/3.9.4/lib/python3.9/unittest/mock.py", line 1337, in patched
    return func(*newargs, **newkeywargs)
  File "/Users/mikolajbalcerek/Dev/athena_alerter/tests/notificators/test_hard_threshold_notificator.py", line 22, in test_handle_batch_event_user_and_channel_value_above_threshold
    sut.handle_batch_event(events)
  File "/Users/mikolajbalcerek/Dev/athena_alerter/bin/notificators/hard_threshold_notificator.py", line 25, in handle_batch_event
    self.handle_single_event(body=record['body'])
  File "/Users/mikolajbalcerek/Dev/athena_alerter/bin/notificators/hard_threshold_notificator.py", line 35, in handle_single_event
    self.send_slack_notification(query, is_send_to_user, is_send_to_admin_channel)
  File "/Users/mikolajbalcerek/Dev/athena_alerter/bin/notificators/hard_threshold_notificator.py", line 52, in send_slack_notification
    self.send_slack_to_channel(text + text_additional_1)
TypeError: can only concatenate str (not "Mock") to str

----------------------------------------------------------------------
Ran 8 tests in 0.016s
```